packages/ciscospark/src/ciscospark.js

```diff
@@ -77,14 +88,32 @@ const ciscospark = new CiscoSpark({
  * })
  * ```
  * Previous versions of the sdk suggested you should pass credentials here.
- * While that still possible, we no longer document it because you should rely
- * on the storage layer instead.
+ * While that still is possible, we no longer document it because you should
+ * rely on the storage layer instead.
+ * @emits CiscoSpark.events:error
  * @param {Object} attrs
+ * @param {Object} attrs.jwt (optional)
+ * @param {Object} attrs.requestJWT (optional)
  * @param {Object} attrs.config (optional)
  * @memberof CiscoSpark
  * @returns {CiscoSpark}
  */
 ciscospark.init = function init(attrs) {
+  if (attrs.jwt) {
+    const jwt = attrs.jwt;
+    const requestJWT = attrs.requestJWT;
+    const etc = omit(attrs, `jwt`, `requestJWT`);
```

Comment:
We could use some fun new syntax features here if we want:

```js
const {
  jwt,
  requestJWT,
  ...etc
} = attrs;
```

Up to you, of course :)